easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-2.4.0-intel-2015b-eb-deps-Python-2.7.10.eb

```diff
@@ -0,0 +1,52 @@
+name = 'OpenFOAM'
+version = '2.4.0'
+pyver = '-Python-2.7.10'
+versionsuffix = '-eb-deps%s' % pyver
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a free, open source CFD software package. 
+ OpenFOAM has an extensive range of features to solve anything from complex fluid flows
+ involving chemical reactions, turbulence and heat transfer, 
+ to solid dynamics and electromagnetics."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+source_urls = ['http://downloads.sourceforge.net/foam/%(version)s']
+sources = [
+    SOURCE_TGZ,
+    'ThirdParty-%(version)s.tgz',
+]
+
+# fix for cyclic symlink issue, which may cause unpacking to fail
+# see also http://www.openfoam.org/mantisbt/view.php?id=1191
+# fixed in recent versions: https://github.com/OpenFOAM/OpenFOAM-2.3.x/commit/f7a485069c778495cc39b308580289f6c2d47163
+unpack_options = "--exclude=*tutorials/mesh/foamyHexMesh/mixerVessel/system/backgroundMeshDecomposition"
+unpack_options += " --exclude=*tutorials/mesh/foamyHexMesh/mixerVessel/system/cellShapeControlMesh"
```

Comment:
see the tag `OpenFOAM-2.4.0` at https://github.com/OpenFOAM/OpenFOAM-2.3.x/commit/f7a485069c778495cc39b308580289f6c2d47163

so, let's drop this here